products/eks.md

```diff
@@ -23,48 +23,54 @@ auto:
 -   custom: true
 
 # EOL dates can be found on https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar
+# Latest release derived from tags in eks-distro Github repo https://github.com/aws/eks-distro
 releases:
+-   releaseCycle: "1.28"
+    releaseDate: 2023-09-01
+    eol: 2024-11-01
+    latestReleaseDate: 2023-09-01
+    latest: '1.28-eks-1'
 -   releaseCycle: "1.27"
     releaseDate: 2023-05-24
     eol: 2024-07-01
     latestReleaseDate: 2023-08-30
-    latest: '1.27-eks-5'
+    latest: '1.27-eks-11'
 
 -   releaseCycle: "1.26"
     releaseDate: 2023-04-11
     eol: 2024-06-01
     latestReleaseDate: 2023-08-30
-    latest: '1.26-eks-6'
+    latest: '1.26-eks-17'
 
 -   releaseCycle: "1.25"
     releaseDate: 2023-02-21
     eol: 2024-05-01
     latestReleaseDate: 2023-08-30
-    latest: '1.25-eks-7'
+    latest: '1.25-eks-21'
```

Comment:
Strange, versions on https://docs.aws.amazon.com/eks/latest/userguide/platform-versions.html does not look in sync with https://github.com/aws/eks-distro/tags. @captn3m0 could you have a look ?

Reply:
Thank you @marcwrobel and @captn3m0.